docs: updated readme (#513)
distributeaid/next-website-v2: README.md

3.Open <http://localhost:3000> with your browser to see the result.
You can start editing the page by modifying app/page.tsx. The page auto-updates as you edit the file.

## Choosing an Issue

Choose an unassigned issue from the [Issue's page on Github](https://github.com/distributeaid/next-website-v2/issues) or see which issues are `Ready for Work` in the [DA Project Board](https://github.com/orgs/distributeaid/projects/15). Note: issues labeled `Good first issue` are ideal for newcomers. Once chosen, be sure to assign yourself to the issue and mark the issue as `In Progress`.

> 

Reviewer: I can't speak for how it should work on this website repo, but on the docs repo I'd be inclined to effusively invite folks to interact/join the team but not require it. 😊

## Branching & Workflows

> We'll have a long-running branch(**saga**) and short-lived branches that will be used to develop parts of the website. An example of a short-lived branch would be feat/about-us. Always ensure you're working on a seperate branch to avoid merge conflicts.

Reviewer: **typo**: tense agreements and a missing space:

```suggestion
We have a long-running branch (**saga**) and create short-lived branches to develop parts of the website. An example of a short-lived branch would be `feat/about-us`. Always ensure you're working on a separate branch to avoid merge conflicts.
```

> We'll have a long-running branch(**saga**) and short-lived branches that will be used to develop parts of the website. An example of a short-lived branch would be feat/about-us. Always ensure you're working on a seperate branch to avoid merge conflicts.

Reviewer: Those are great ideas!

## Coding Conventions / Best Practices

- Try to avoid using raw #hexadecimal values. View `/src/stylesheets/` or `tailwind.config.ts` to see color value names.

> - Try to use Radix Ui's component library and use tailwind Css for supplimentary styling.

Reviewer: I agree with Kai about not needing the "Try to" language, and there are a couple of little case inconsistencies and a typo:

```suggestion
- Use Radix UI's component library and use tailwind CSS for supplementary styling.
```

## Crafting commit messages

> When crafting commit messages, try to group commits that are related to a particular topic. For example, if you're making commits related to the styling, only include files that are related to styling. This makes it easier to read through the commits and see the exact files that made changes to the styles. See [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) for more information.

Reviewer: I think it definitely makes sense to put it in here for now, and if we decide to centralize this kind of info in future we can take it out. 😊

## Crafting a Pull Request (PR)

> - Include the purpose of the Pull Request and link the issue related to the PR (with the `#` sign)

Reviewer: > **question:** should we also add that they should add a reviewer or would one assume that to be logical (I tend towards over explaining and assuming the worst user lol)

Nope! Not all projects ask PR authors to assign reviewers, and prefer to assign reviewers themselves. So if the DA convention is for the author to assign a reviewer, then we should say so (and tell them who to assign!).

> - Include the purpose of the Pull Request and link the issue related to the PR (with the `#` sign)

Reviewer 2: > Exactly! This repo isn't under my jurisdiction, but the teams idea would be great to bring up to @jtfairbank!

Just to clarify, I meant team aspect as in that we run our team a little differently than most OSS teams, i.e. sync and not just async. But perhaps you’re on to something here: we have a Maintainer team, we could tag that team or even create a team just for reviews. This is a TBD to bring up at a sprint meeting or on slack, imho. (probably slack is better because async means more folks can take part)

Happy coding!!!

# Advanced

## Creating Issues

> On the [Issue's page on Github](https://github.com/distributeaid/next-website-v2/issues), click the green `New Issue` button or this [New Issue Link](https://github.com/distributeaid/next-website-v2/issues/new). Add a descriptive title with an optional `[type tag]` such as [component] or [page]. In the description body summarize the issue. Add external links like such as to figma designs, if applicable. Add parts of the codebase where changes would be made. You can assign the issue to yourself or someone else, or leave for another person to pick it up. Add appropriate labels and types. Then add the issue to the project **DA Tech**.

Reviewer: **typo**: extraneous apostrophe:

```suggestion
On the [Issues page on Github](https://github.com/distributeaid/next-website-v2/issues), click the green `New Issue` button or this [New Issue Link](https://github.com/distributeaid/next-website-v2/issues/new). Add a descriptive title with an optional `[type tag]` such as [component] or [page]. In the description body summarize the issue. Add external links like such as to figma designs, if applicable. Add parts of the codebase where changes would be made. You can assign the issue to yourself or someone else, or leave for another person to pick it up. Add appropriate labels and types. Then add the issue to the project **DA Tech**.
```

## Reviewing Pull Requests

When reviewing changes, try to first give praise for the work done. Leave a general summary of the changes needed to be made. Individual comments can be put to show specific instances or examples directly in the codebase. See [Conventional Comments](https://conventionalcomments.org/) for more information.

A good comment will ideally have the following attributes:

- Label => Signifies what kind of comment is being left
- Subject => Main message of the comment
- Decorations (_optional_) => These are extra decorating labels for the comment. They are surrounded by parentheses and comma-separated.
- Discussion (_optional_) =>This contains supporting statements, context, reasoning, and anything else to help communicate the “why” and “next steps” for resolving the comment.

Labeling comments encourages collaboration and saves hours of undercommunication and misunderstandings.

## Merging Branches

Use [`git rebase`](https://www.atlassian.com/git/tutorials/rewriting-history/git-rebase) to resolve any merge conflicts between branches.

> ## License

Reviewer: **question**: Should there be a license here?
